README.md

```diff
@@ -85,6 +85,9 @@ git clone https://github.com/purnima143/Kurakoo.git
     npm run server
     ```
 
+    Link for the screenshots, how to setup backend locally:
+    https://docs.google.com/document/d/1Ofx0v6mXjIhNQ0ddYh7btWBZWjqgNobQffi3E1o5Lkw/edit?usp=sharing
```

Comment:
It would be great if you attach a screenshot in the new folder `assets/Backend/screenshot` and you can point here that screenshot, rather than create any new docs file.

Reply:
okay